spec/criteria_spec.cr

```diff
@@ -9,41 +9,45 @@ end
 describe LuckyRecord::Criteria do
   describe "is" do
     it "uses =" do
-      age.is(30).to_sql.should eq ["SELECT * FROM users WHERE age = $1", "30"]
+      age.is(30).to_sql.should eq ["SELECT #{query_me_columns} FROM users WHERE age = $1", "30"]
     end
   end
 
   describe "is_not" do
     it "uses !=" do
-      age.is_not(30).to_sql.should eq ["SELECT * FROM users WHERE age != $1", "30"]
+      age.is_not(30).to_sql.should eq ["SELECT #{query_me_columns} FROM users WHERE age != $1", "30"]
     end
   end
 
   describe "gt" do
     it "uses >" do
-      age.gt("30").to_sql.should eq ["SELECT * FROM users WHERE age > $1", "30"]
+      age.gt("30").to_sql.should eq ["SELECT #{query_me_columns} FROM users WHERE age > $1", "30"]
     end
   end
 
   describe "gte" do
     it "uses >=" do
-      age.gte("30").to_sql.should eq ["SELECT * FROM users WHERE age >= $1", "30"]
+      age.gte("30").to_sql.should eq ["SELECT #{query_me_columns} FROM users WHERE age >= $1", "30"]
     end
   end
 
   describe "lt" do
     it "uses <" do
-      age.lt("30").to_sql.should eq ["SELECT * FROM users WHERE age < $1", "30"]
+      age.lt("30").to_sql.should eq ["SELECT #{query_me_columns} FROM users WHERE age < $1", "30"]
     end
   end
 
   describe "lte" do
     it "uses <=" do
-      age.lte("30").to_sql.should eq ["SELECT * FROM users WHERE age <= $1", "30"]
+      age.lte("30").to_sql.should eq ["SELECT #{query_me_columns} FROM users WHERE age <= $1", "30"]
     end
   end
 end
 
 private def age
   QueryMe::BaseQuery.new.age
 end
+
+private def query_me_columns
```

Comment:
I think since this is repeated a bit, I think it might be good to add a `COLUMNS` constant to the QueryMe class so you can do `QueryMe::COLUMNS` instead of defining this helper in multiple specs. What do you think?

Reply:
`QueryMe` is defined multiple times with slight differences, so cannot quite remove it. But I agree, a `COLUMNS` puts it at a better place. Done ✅